lib/vsc/install/headers.py

```diff
@@ -117,6 +117,21 @@ def get_header(filename, script=False):
     return header, shebang
 
 
+def gen_license_header_brussel(license_template):
+    """
+    This adjusts the license template for use in Brussel
+    :param license_template: a license template
+    """
+    # replace university name
```

Comment:
no, template it for all sites (using templating ;)

Reply:
done, but it will now fail if the url doesn't contain either `hpcugent` or `sisc-hpc`. Should I by default fallback to `hpcugent`?

Reply:
hmmmm, lets try without the fallback